ledger/sandbox/BUILD.bazel

```diff
@@ -316,47 +319,54 @@ server_conformance_test(
     ],
 )
 
+# This test asserts that the Daml engine is configured correctly for the sandbox executable.
+# It does not depend on the ledger backend, that's why it runs on a single server.
```

Comment:
When Sandbox-on-X is ready, we could move the tests that don't depend on the ledger backend there.

Reply:
Each participant executable needs one of these tests. It checks that the Daml engine configuration (that is hardcoded in the participant application) corresponds to the ledger backend type.

Reply:
I see. I was assuming that the assertion is static, but indeed I see that `sandbox-classic` does not forbid V0 contract ids. Thanks for clarifying